SWLOR.Game.Server/Service/Death.cs

```diff
@@ -28,10 +28,27 @@ public static void OnPlayerDeath()
                 factionMember = GetNextFactionMember(hostile, false);
             }
 
-            const string RespawnMessage = "You have died. You can wait for another player to revive you or respawn to go to your home point.";
-            PopUpDeathGUIPanel(player, true, true, 0, RespawnMessage);
+            if (GetLocalBool(hostile, "SUBDUAL_MODE"))
```

Comment:
As mentioned in another comment, I'd recommend doing a check here for the DB entity's SubdualMode instead of a local variable.

Reply:
Done